client/packages/host/src/Admin/ServerSettings.tsx

```diff
@@ -79,6 +88,34 @@ export const ServerSettings = () => {
           </>
         }
       />
+
+      <Setting
+        title={t('label.download-database')}
+        component={
+          <>
+            <LoadingButton
+              disabled={databaseSettings?.databaseType !== DatabaseType.SqLite}
```

Comment:
If the button is going to show, it would be helpful to know why the button is disabled for me.
Either helper text below the `Download Database` heading, or a tooltip, or the preferred option of the PM/design team (according to #1912) - the button is enabled but alerts when you click it. 

That latter option is something I think none of the developers want, btw. Some help text?

Reply:
Definitely a tricky area with differing opinions on how to handle it. I did try to put in a title but doesn't show for disabled buttons. Another option is to take out the check entirely, then you get a backend error if you are on postgres. I nearly did that originally. I personally like tooltips but they're not so great on android...

Anyway, I've found I can get tooltips working by adding span element in, I think that will do for now?

Reply:
yep - that's good for me, thanks!